types/stats.js/stats.js-tests.ts

```diff
@@ -1,17 +1,21 @@
-
-
-var stats = new Stats();
+const stats = new Stats();
 stats.showPanel( 1 ); // 0: fps, 1: ms, 2: mb, 3+: custom
 document.body.appendChild( stats.dom );
 
+const panel = stats.addPanel(
```

Comment:
@andy-ms I added `// $ExpectType Stats.Panel` above this line, but got the following error:

```
$ dtslint types stats.js      
/home/danvk/github/DefinitelyTyped/types/stats.js/stats.js-tests.ts
ERROR: 6:1  expect  Expected type to be:
  Stats.Panel
got:
  any
```

whereas in vscode with TypeScript 2.5.2,

![hover type](https://user-images.githubusercontent.com/98301/33044735-0b570c4e-ce18-11e7-8475-bb43085ec0a0.png)

does this match your expectations?

Reply:
Looks like a bug - try again. (Microsoft/dtslint#90)